core/dbt/include/global_project/macros/unit_test_sql/get_fixture_sql.sql

```diff
@@ -67,7 +68,7 @@ union all
 {%- for column_name, column_value in row.items() -%}
 {% set row_update = {column_name: column_value} %}
 {%- if column_value is string -%}
-{%- set row_update = {column_name: safe_cast(dbt.string_literal(column_value), column_name_to_data_types[column_name]) } -%}
+{%- set row_update = {column_name: safe_cast(dbt.string_literal(dbt.escape_single_quotes(column_value)), column_name_to_data_types[column_name]) } -%}
```

Comment:
So that it's:
```sql
select  cast(parse_json('[\'a\',\'b\',\'c\']') as ARRAY) AS tested_column
```
Rather than:
```
 select cast(parse_json('['a','b','c']') as ARRAY) AS tested_column
```